bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAlarmHandler.java

```diff
@@ -218,6 +225,7 @@ private void updateZone(Alarm msg) {
             case ZONE_ALARM_TECHNICAL:
             case ZONE_ALARM_TECHNICAL_RESET:
                 updateZoneAlarm(w);
+                updateZoneAlarmTimeStamp();
```

Comment:
you have to choose when updateZoneAlarmTimeStamp is called. This way it's called 2 times: in the switch inside updateZoneAlarm and then again in the updateZone switch

Reply:
ok I will call it in updateZone() only